doc/installing.rst

```diff
@@ -52,8 +52,8 @@ create a local environment and install `joblib` in it. One of the pros of
 this method is that you never have to become administrator, and thus all
 the changes are local to your account and easy to clean up.
 
-* **If you are under Pyton 2.6 or above**
-  
+* **If you are under Python 2.6 or above**
```

Comment:
> This section can be removed since we only support python 2.6+ and python 3.3+ according to README.rst.

I guess just remove the "If you are under ..." mention instead.